### PR TITLE
Replace broken link in G56

### PR DESCRIPTION
**Pull request by hidde-logius**
The page this links no longer exists.

### CODE REVIEW

**Comment by scottaohara** [comment]
the link is still available via the wayback machine - https://web.archive.org/web/20170716020055/http://www.eramp.com/david/audio_contrast_general_techs.htm

just posing the question of should that be linked to instead, or is deleting the link all together fine in this instance?

**Comment by hidde-logius** [comment]
Fair enough, think the wayback URL is more useful than nothing. When I looked, I did not find a different page with similar content either, so seems like this is the best we can do.

File: techniques/general/G56.html [hunk 1]
```diff
@@ -68,13 +68,12 @@ <h3>Insufficient Audio Contrast between a narrator and background music</h3>
          <ul>
             <li>#5 is true</li>
          </ul>
-      </section>
```
Comment (kfranqueiro) [inline techniques/general/G56.html:71]: It looks like the inner end section tag (corresponding to Expected Results) was removed in this change?

(As far as I can tell, this should not change; there should still be 2 end section tags here, for Expected Results and Tests)

File: techniques/general/G56.html [hunk 2]
```diff
@@ -68,13 +68,12 @@ <h3>Insufficient Audio Contrast between a narrator and background music</h3>
...
-   </section><section id="resources"><h2>Resources</h2>
-      
+      </section><section id="resources"><h2>Resources</h2>
```
Comment (patrickhlauke) [inline techniques/general/G56.html:71]:
```suggestion
      </section>
    </section><section id="resources"><h2>Resources</h2>
```

File: techniques/general/G56.html (hunk 1 above)
Comment (patrickhlauke) [reply techniques/general/G56.html:71]: confirmed by looking at this in validator. made the fix

**Comment by hidde** [comment]
Thanks, both!

**Pull request merged**